.github/workflows/elixir.yml

```
@@ -64,10 +64,10 @@ jobs:
         key: ${{ runner.os }}-build-${{ hashFiles(format('{0}{1}', github.workspace, '/mix.lock')) }}
         restore-keys: |
           ${{ runner.os }}-build-
-    - uses: actions/setup-elixir@v1
+    - uses: erlef/setup-beam@v1
       with:
-        otp-version: 22.3
-        elixir-version: 1.10.4
+        otp-version: 23.3.4
+        elixir-version: 1.11.4
```

In case of always wanting to use latest possible patch:

```suggestion
        elixir-version: '1.11'
```